### ISSUE TITLE
False positive related to prefix notation

### ISSUE THREAD

**Issue by myint** [L2Program/FlintPlusPlus#7]
The prefix notation provides no benefit for primitive types. At the very least this false positive is noise. Worse, the advise is wrong in cases where changing it results in changing program behavior.

`foo.cc`:

``` cpp
int main()
{
    int x = 0;
    x++;
}
```

```
$ ./flint++ foo.cc
===============================================================================
File foo.cc:
Errors:   0
Warnings: 0
Advice:   1
===============================================================================

Line 4: Advice

Use prefix notation '++x'.

Postfix incrementers inject a copy operation, almost doubling the workload.
```
